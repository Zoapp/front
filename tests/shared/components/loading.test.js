import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '@shared/components/loading';

describe('Loading', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Loading />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
