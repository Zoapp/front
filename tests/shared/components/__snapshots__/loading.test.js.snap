// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<div
  className="mdc-card"
  style={
    Object {
      "margin": "auto",
      "width": "512px",
    }
  }
>
  <section
    className="mdc-card__primary"
  >
    
    <h1
      className="mdc-card__title"
    >
      Processing
    </h1>
    
  </section>
  <section
    className="mdc-card__supporting-text"
  >
    Please wait ...
  </section>
  <section
    className="mdc-card__actions"
  />
</div>
`;
